### DISCUSSION TITLE
Standard website footer

### DISCUSSION THREAD

**Discussion by emmanuelbernard** [commonhaus/foundation#12] [Q&A]
https://github.com/commonhaus/foundation-draft/blob/main/governance/ip-policy-guidance.md#1-licensing

mentions a mandatory website footer
Why?
What would be in it?

**Comment by ebullient** [comment]
Something like [this](https://github.com/commonhaus/foundation-draft/blob/main/governance/website-footer.md), but I was hoping to hammer that out with whoever the initial bootstrapping council members are.

**Comment by sigrunixia** [comment]
Note that some locales require such a footer, such as the German [Impressum](https://www.firma.de/en/company-formation/impressum-what-to-include-in-a-company-legal-disclosure-in-germany-including-free-template/).

**Comment by kenfinnigan** [comment]
Closing this discussion as resolved
